test/functional/mempool_accept.py

```diff
@@ -91,20 +92,22 @@ def run_test(self):
         tx.deserialize(BytesIO(hex_str_to_bytes(raw_tx_0)))
         txid_0 = tx.rehash()
         self.check_mempool_result(
-            result_expected=[{'txid': txid_0, 'allowed': True}],
+            result_expected=[{'txid': txid_0, 'allowed': True, 'vsize': tx.get_vsize(), 'fees': {'base': Decimal(str(fee))}}],
```

Comment:
unrelated nit: What happens if `fee` was made a `Decimal` as opposed to `float` when defined?